external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -88,26 +89,53 @@ bool isElementwiseOp(Operation *op) {
   // clang-format on
 }
 
+bool isFusibleOp(Operation *op) {
```

Comment:
Separating "element-wise" from "other ops we want to fuse."  And "fusible" is the dictionary spelling, though I keep thinking of "fusible links."

Reply:
We could do some good-old fashioned programmer spelling and say fusable anyway

Reply:
As a kid, I competed in spelling bees.  "Fusable" would make me twitch.  Though I did consider "fuseable."

Reply:
Entirely fair. I'd argue for "fuseable", as it doesn't send people running to a dictionary for a form that's not super-recognized.

(and it's neat to learn about the spelling bee thing!)